extensions/response.py

```
@@ -31,12 +32,15 @@ def load(self, data: dict | int | float | str | bool | list | None) -> None:
             for index in range(len(data)):
                 data[index] = self.convert(data[index])
         self.data = data
-    def convert(self, value: any) -> dict | int | float | str | bool | list | None:
+    # CLASS -> CONVERT
+    def convert(self, value: Any) -> dict | int | float | str | bool | list | None:
```

Ruff (0.12.2)

`Any` may be undefined, or defined from star imports

(F405)